.github/workflows/cd.yml

```
@@ -0,0 +1,20 @@
+       steps:
+          - name: Check out code
+            uses: actions/checkout@v6
+  
+          - name: Set up Go
+            uses: actions/setup-go@v6
+            with:
+              go-version: "1.27.1"
+  
+          - name: Build
+            run: ./scripts/buildprod.sh
+  '
```

actionlint (1.7.12)

[error] could not parse as YAML: found unexpected end of stream

(syntax-check)